### PR TITLE
Update 2015-10-17-release-notes.md

### PR DESCRIPTION
**Pull request by lblackstone**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-5334241d50/).

**Comment by thomasem** [comment]
:+1:

File: _reference/2015-10-17-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ topics:
   - docker
 ---
 
+### April 21, 2016
+
+#### What's new
+
+* Patched EOF bug in Docker Engine 1.10.3
```
Comment (everett-toews) [inline _reference/2015-10-17-release-notes.md]: Link to https://github.com/getcarina/feedback/issues/65 ?

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-c7c2bab513/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-8a9be7c5b9/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-8124e27b51/). :bow:

**Pull request merged**